public/dataTypes.js

```diff
@@ -189,7 +197,18 @@ List.prototype.renderChart = function (doParent) {
         var category = this.library.getCategoryById(this.categoryIds[i]);
         if (category) {
             var points = {};
-            var categoryTotal = category.subtotal;
+
+            var categoryTotal;
+            if (type === 'lpConsumableWeight') {
```

Comment:
Along with the class to data attribute change,  we can rename these to not use class names and instead just  "CONSUMABLE" etc